website/src/js/components/SearchBar.js

```diff
@@ -105,12 +106,12 @@ class SearchBar extends Component {
    * @description Pushes the given active search string to the browser history
    */
 
-  navigateToSearch(active_search) {
+  navigateToSearch(active_search, fields ='all') {
 
     const active_sort = activeSortOption(this.props.options);
     const route = formSearchRoute({
-      // FIXME: https://github.com/iodepo/OceanBestPractices/issues/199
-      active_fields: activeFieldsString(this.props.fields),
+      active_field: activeFieldsString(this.props.fields),
```

Comment:
I realize here that if you change these to camelCase you'll have to edit a different file. Let's just make changes to this file for now and leave the snake_case here. We can update the other file later.